packages/@aws-cdk/toolkit-lib/lib/api/garbage-collection/garbage-collector.ts

```diff
@@ -178,6 +178,14 @@ interface GarbageCollectorProps {
    * @default true
    */
   readonly confirm?: boolean;
+
+  /**
+   * Non-CDK stack names or glob patterns to skip when encountering unauthorized access errors during garbage collection.
+   * You must explicitly specify non-CDK stack names - CDK stacks will be rejected to prevent accidental asset deletion.
+   *
+   * @default undefined
+   */
+  readonly skipUnauthorizedStacksWhenNonCdk?: string[];
```

Comment:
we need to workshop this name. how about:

```ts
{
  readonly nonCdkStacksToSkip?: string[];
}
```

or 

```ts
{
  readonly nativeCloudFormationStacksToSkip?: string[];
}
```

It's very important to get this right. @mrgrain do you have ideas?

Reply:
i think, while the original user had an unauthorized stack they wanted to skip, the real importance of this property is that it must must must not be a cdk stack. my preference is still `nativeCloudFormationStacksToSkip`. Please follow up with @mrgrain to make a determination here.